en/docs/consuming-services/generate-an-access-token.md

```diff
@@ -1,43 +1,58 @@
-# Generate an Access Token
+# Generate an access token
 
-Using access tokens for request authorization strengthens security measures, particularly in preventing certain types of denial-of-service (DoS) attacks aimed at published APIs. API consumers generate access tokens to access the API, incorporating them into their HTTP header requests as simple string values.
+Using access tokens for request authorization enhances security by preventing certain types of denial-of-service (DoS) attacks on published APIs. API consumers generate access tokens to access APIs, including them as string values in HTTP header requests.
 
-When you register an application on the Developer Portal, you can generate a consumer key and a consumer secret for it. The consumer key and the consumer secret represent the credentials of the application. Similar to a user's username, the consumer key becomes the unique identifier of the application so that you can use it to authenticate the request to the API. Choreo issues an access token for the application against the mentioned consumer key.
+When you register an application in the Developer Portal, you can generate a consumer key and consumer secret. These credentials represent the application's identity. The consumer key acts as the unique identifier for the application, similar to a username, and is used to authenticate API requests. Choreo issues an access token for the application based on the consumer key.
 
-This section walks you through the steps to generate an access token for your application in Choreo.
-
-Let's get started!
+This guide walks you through the steps to generate an access token for your application in Choreo.
 
 ## Prerequisites
 
-Before you try out this guide, be sure you have the following:
+Before proceeding, ensure you have the following:
+
+1. An application in the [Choreo Developer Portal](https://devportal.choreo.dev). If you don’t have one, [create a new application](https://wso2.com/choreo/docs/consuming-services/manage-application/#step-1-create-an-application).
+2. [Generate keys for the application](https://wso2.com/choreo/docs/consuming-services/create-an-application/#step-2-generate-keys).
+3. [Subscribe APIs to the application](https://wso2.com/choreo/docs/consuming-services/create-a-subscription/#manage-subscriptions).
+
+## Generate an access token via curl
 
-1. An application in the [Choreo Developer Portal](https://devportal.choreo.dev). If you do not have an application created, [create a new application](https://wso2.com/choreo/docs/consuming-services/manage-application/#step-1-create-an-application)
-2. [Generate keys for the application](https://wso2.com/choreo/docs/consuming-services/create-an-application/#step-2-generate-keys). 
-3. [Subscribe APIs to the application](https://wso2.com/choreo/docs/consuming-services/create-a-subscription/#manage-subscriptions). 
+Follow these steps to generate an access token for your application using cURL:
 
+1. **Go to Applications**  
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. Any reason to do this? https://developers.google.com/style/text-formatting